Modules/Loadable/Sequences/Widgets/qMRMLSequenceBrowserPlayWidget.h

```diff
@@ -44,6 +44,10 @@ class Q_SLICER_MODULE_SEQUENCES_WIDGETS_EXPORT qMRMLSequenceBrowserPlayWidget
   Q_OBJECT
   QVTK_OBJECT
 
+  /// Enable or disable recording control buttons when recording
+  Q_PROPERTY(bool recordingControlsVisible READ recordingControlsVisible WRITE setRecordingControlsVisible STORED true)
```

Comment:
No need to set STORED (it is stored by default).

```suggestion
  Q_PROPERTY(bool recordingControlsVisible READ recordingControlsVisible WRITE setRecordingControlsVisible)
```